Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -309,6 +309,9 @@ public class NavigatorIndex {
         case languageGroup = 127
         case container = 254
         case groupMarker = 255 // UInt8.max
+
+        // C++ symbols
+        case namespace = 48
```

Comment:
Just for how these enum cases are organized, please add this case _above_ the special items so that the cases continue to be defined in the order of their raw value.

Reply:
I also believe that this is a source breaking change since any existing code that was switching over this public enum now has a new case that it would need to handle. Maybe there's nothing we can do about it but it's worth calling mentioning.

Reply:
> I also believe that this is a source breaking change since any existing code that was switching over this public enum now has a new case that it would need to handle. Maybe there's nothing we can do about it but it's worth calling mentioning.

Are you aware of any way to do this in a non source breaking manner?

Reply:
Unfortunately I don't know of any other way. If we need to make a source breaking change it might be worth discussing if we should make a larger change now so that future changes can be made in a non-source breaking manner.